tests/RuntimeTest.php

```diff
@@ -142,6 +142,30 @@ public function testGetVendorUrlReturnsPhpDotNetWhenRunningPhp(): void
         $this->assertSame('https://secure.php.net/', $this->env->getVendorUrl());
     }
 
+    public function testDiscardingCommentsReturnsFalseIfOpcacheIsNotActive(): void
```

Comment:
This test looks unrelated to the topic of the PR. What am I missing?

Reply:
Oh, I noticed that this part was uncovered in the coverage so I just added it. I can remove it so that the tests are related only to the above change 👍🏼

Reply:
Yes, please. As it is, this test might fail depending on the environment (pun intended).